modules/weko-items-ui/weko_items_ui/views.py

```diff
@@ -243,46 +243,66 @@ def get_json_schema(item_type_id=0, activity_id=""):
     :param activity_id: Activity ID.  (Default: Null)
     :return: The json object.
     """
+    def set_validation_message(item, cur_lang):
+        """Set validation message.
+
+        :param item: json of control (ex: json of text input).
+        :param cur_lang: current language.
+        :return: item, set validationMessage attribute for item.
+        """
+        i18n = 'validationMessage_i18n'
+        message_attr = 'validationMessage'
+        if i18n in item:
+            item[message_attr] = item[i18n][cur_lang]
+
+    def translate_validation_message(item_property, cur_lang):
+        """Recursive in order to set translate language validation message.
+
+        :param item_property: .
+        :param cur_lang: .
+        :return: .
+        """
+        items_attr = 'items'
+        properties_attr = 'properties'
+        if items_attr in item_property:
```

Comment:
Change 
`if items_attr in item_property:`
to
`if isinstance(item_property, dict) and isinstance(item_property.get(items_attr), dict): `